src/components/SideNav.jsx

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { bool } from 'prop-types';
 import {
   settingsIcon,
   historyIcon,
   readingStatsIcon,
   bookmarkIcon
 } from '../assets/img__func/icons_svg';
 
+
 /**
  * @description side nav class component
  * @returns {undefined}
  */
-const SideNav = () => (
-  <aside className="side-nav-parent">
+export const SideNav = ({ isLoggedIn }) => (
+  <aside className="side-nav-parent" style={isLoggedIn !== false ? { top: '91.59px' } : { top: '900000000000000000em' }}>
```

Comment:
Can you please clarify what this line does?

Reply:
It's an intentional temporary inline style to hide the Side nav bar if the user isn't signed in. It will be removed when the pages are functional

Reply:
Okay.